File: LATIHAN2/Latihan2_Java/Product.java
public class Product {

	private int price;
	private String idProduct;

	// empty constuctors
	Product() {

	}

	// constructors
	Product(int price, String idProduct) {
		this.price = price;
		this.idProduct = idProduct;
	}

	// getter and setter
	public void setPrice(int price) {
		this.price = price;
	}

	public int getPrice() {
		return price;
	}

	public void setIdProduct(String idProduct) {
		this.idProduct = idProduct;
	}

	public String getIdProduct() {
		return idProduct;
	}

}

File: LATIHAN2/Latihan2_PHP/Product.php
<?php

// define product class
class Product
{
	// class attribute
	private $idProduct;
	private $price;

	// empty constructors
	public function Product($idProduct = "", $price = 0)
	{
		$this->idProduct = $idProduct;
		$this->price = $price;
	}

	// getter and setter
	public function setIdProduct($idProduct)
	{
		$this->idProduct = $idProduct;
	}

	public function getIdProduct()
	{
		return $this->idProduct;
	}

	public function setPrice($price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->price;
	}
}
